From 843bcf23ce6ef6ab24d956ad6d482c2d5d1b31a7 Mon Sep 17 00:00:00 2001
From: Andy Yan <andy.yan@rock-chips.com>
Date: Mon, 14 Aug 2017 16:37:48 +0800
Subject: [PATCH] UPSTREAM: dt-bindings: pwm: Add description for rv1108 PWM

Add device tree bindings document for PWM on Rockchip rv1108 SoC.

Change-Id: I3da3c47c5755f4673e4007fe5b708295f2d3e8bd
Signed-off-by: Andy Yan <andy.yan@rock-chips.com>
Acked-by: Rob Herring <robh@kernel.org>
Signed-off-by: Thierry Reding <thierry.reding@gmail.com>
Signed-off-by: Sugar Zhang <sugar.zhang@rock-chips.com>
(cherry picked from commit 226d1005dbad77f1587ea4cfd151b15cd0a17da1)
---
 Documentation/devicetree/bindings/pwm/pwm-rockchip.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/pwm/pwm-rockchip.txt b/Documentation/devicetree/bindings/pwm/pwm-rockchip.txt
index bb2b8a3df68a..b8094e865aa9 100644
--- a/Documentation/devicetree/bindings/pwm/pwm-rockchip.txt
+++ b/Documentation/devicetree/bindings/pwm/pwm-rockchip.txt
@@ -3,7 +3,8 @@ Rockchip PWM controller
 Required properties:
  - compatible: should be "rockchip,<name>-pwm"
    "rockchip,rk2928-pwm": found on RK29XX,RK3066 and RK3188 SoCs
-   "rockchip,rk3288-pwm": found on RK3288 SoC
+   "rockchip,rk3288-pwm": found on RK3288 SOC
+   "rockchip,rv1108-pwm", "rockchip,rk3288-pwm": found on RV1108 SoC
    "rockchip,vop-pwm": found integrated in VOP on RK3288 SoC
    "rockchip,rk3328-pwm": found on RK3328 SoC
    "rockchip,rk3399-pwm", "rockchip,rk3288-pwm": found on RK3399 SoC
-- 
2.35.3

